commit 8c681864a96e97adf7b6cdd08618783772751b13 (refs/changes/32/3732/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-03-28T10:47:48-07:00 (1 year, 6 months ago)
    
    TRITON-282 drop obsolete gsed'ing of IMGAPI SMF manifest at zone setup
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/boot/setup.sh b/boot/setup.sh
index 565a253..8f1fb0c 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -40,7 +40,6 @@ if [[ ! -d $LOCAL_BASE_DIR ]]; then
 fi
 chown nobody:nobody $LOCAL_BASE_DIR
 
-$(/opt/local/bin/gsed -i"" -e "s/@@PREFIX@@/\/opt\/smartdc\/imgapi/g" /opt/smartdc/imgapi/smf/manifests/imgapi.xml)
 /usr/sbin/svccfg import /opt/smartdc/imgapi/smf/manifests/imgapi.xml
 
 # Log rotation.
